demos/src/radios-button-styling.mustache

```diff
@@ -0,0 +1,57 @@
+<div class="o-forms">
+    <p>No default selection</p>
+    <fieldset class="o-forms__radio-buttons">
```

Comment:
https://github.com/Financial-Times/o-forms/blob/master/demos/src/radios.mustache#L1

Maybe something along these lines?
```
<fieldset class="o-forms">
    <legend class="o-forms__label">No default selection</legend>
    <div class="o-forms__radio-buttons-group">
        <input type="radio" name="radio4" value="daily" class="o-forms__radio-buttons__button" id="radio41" />
        <label for="radio41" class="o-forms__label">Daily</label>
        <input type="radio" name="radio4" value="weekly" class="o-forms__radio-buttons__button" id="radio42" />
        <label for="radio42" class="o-forms__label">Weekly</label>
    </div>
</fieldset>
```
`o-forms` is used as a containing element for the field. And the legend becomes the field label visually speaking.

`o-forms__radio-buttons__wrapper` would then become `o-forms__radio-buttons-wrapper`. What would you think about calling it `o-forms__radio-buttons-group` for consistency with the new classes just added in v5?
https://github.com/Financial-Times/o-forms/blob/master/demos/src/radios.mustache#L4

With these changes the `o-forms__radio-buttons` class and styles aren't needed (`o-forms` removes the default legend styling)
https://github.com/Financial-Times/o-forms/pull/183/files#diff-12b0210cae4dd2ece978ad72b68c87c2R152

Reply:
Without any specific wrapper, this feature could then be output along these lines:
https://github.com/Financial-Times/o-forms/blob/master/src/scss/base.scss#L91

Reply:
Great suggestions thanks. All done.